caddy/README.md

```diff
@@ -29,55 +29,112 @@ Here's the things we find most useful:
 caddy file-server --browse --listen :4040
 ```
 
+### How to serve HTTPS on localhost
+
+Caddy can be used to test with https on localhost.
+
+`Caddyfile`:
+
+```Caddyfile
+https://localhost:443 {
+    handle /api/* {
+        reverse_proxy localhost:3000
+    }
+    
+    handle /* {
+        root * ./public/
+        file_server
+    }
+}
+```
+
+```sh
+caddyfile run --config ./Caddyfile
```

Comment:
```suggestion
caddy run
```

Reply:
Same as above. Too much mental load to have multiple ways to do the same thing, and to have the most common option hidden away.